node/src/GlideClient.ts

```diff
@@ -371,6 +372,37 @@ export class GlideClient extends BaseClient {
         return this.createWritePromise(createTime());
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key on `destinationDB`. When `replace`
+     * is true, removes the `destination` key first if it already exists, otherwise performs no action.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * since Valkey version 6.2.0.
+     *
+     * @param source - The key to the source value.
+     * @param destination - The key where the value should be copied to.
+     * @param destinationDB - The alternative logical database index for the destination key.
+     * @param replace - If the destination key should be removed before copying the value to it.
```

Comment:
Andrew has asked me to specify that arguments are optional in the docs and clarify what the default behavior is. Maybe we should clarify as a team whether we want to enforce this going forward? If so, can you change to
```suggestion
     * @param destinationDB - (Optional) The alternative logical database index for the destination key. If not provided, the current database will be used.
     * @param replace - (Optional) If the destination key should be removed before copying the value to it. If not provided, no action will be performed if the key already exists.
```